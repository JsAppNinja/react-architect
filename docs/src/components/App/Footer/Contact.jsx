import React from 'react';
import { Button } from 'react-md';

const Contact = () => (
  <section className="md-cell md-cell--4">
    <h4 className="md-title">Contact</h4>
    <Button flat primary href="mailto:mlaursen03@gmail.com" iconChildren="mail">
      Mikkel Laursen
    </Button>
  </section>
);

export default Contact;
