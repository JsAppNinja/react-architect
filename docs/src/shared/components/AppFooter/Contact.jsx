import React, { PureComponent } from 'react';
import FontIcon from 'react-md/lib/FontIcons';
import FlatButton from 'react-md/lib/Buttons/FlatButton';

export default class Contact extends PureComponent {
  render() {
    return (
      <div className="contact">
        <h4 className="md-title">Contact</h4>
        <FlatButton primary label="Mikkel Laursen" href="mailto:mlaursen03@gmail.com" type={null}>
          <FontIcon>mail</FontIcon>
        </FlatButton>
      </div>
    );
  }
}
