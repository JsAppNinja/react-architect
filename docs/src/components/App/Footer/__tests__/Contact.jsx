/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import Contact from '../Contact';

describe('Contact', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Contact />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
