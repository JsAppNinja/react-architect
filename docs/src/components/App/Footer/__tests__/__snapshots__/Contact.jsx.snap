// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render correctly 1`] = `
<section
  className="md-cell md-cell--4"
>
  <h4
    className="md-title"
  >
    Contact
  </h4>
  <a
    className="md-btn md-btn--flat md-btn--text md-pointer--hover md-text--theme-primary md-ink--primary md-inline-block"
    disabled={undefined}
    href="mailto:mlaursen03@gmail.com"
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
  >
    <div
      className="md-icon-separator"
    >
      <i
        className="md-icon material-icons md-text--inherit"
        style={undefined}
      >
        mail
      </i>
      <span
        className="md-icon-text"
        style={undefined}
      >
        Mikkel Laursen
      </span>
    </div>
  </a>
</section>
`;
